cookbook/entities/deleting_tasks.rst

```diff
@@ -0,0 +1,113 @@
+How to Delete Tasks
+===================
+
+The data grid that you created so far is able to let the user delete entries without having to
```

Comment:
After reading this sentence, i have decide that platform allows to delete the records with reloading the page. Is this true?